tf2/include/tf2/LinearMath/MinMax.h

```diff
@@ -18,25 +18,25 @@ subject to the following restrictions:
 #define GEN_MINMAX_H
 
 template <class T>
-TF2SIMD_FORCE_INLINE const T& tf2Min(const T& a, const T& b) 
+TF2_PUBLIC TF2SIMD_FORCE_INLINE const T& tf2Min(const T& a, const T& b) 
```

Comment:
You shouldn't need to export template symbols. Templates are getting evaluated on caller side, meaning they are always in the according compilation unit.